lib/generators/ninetails/seed/templates/page_seed.rb

```diff
@@ -0,0 +1,9 @@
+Ninetails::Seeds::Generator.generate_page do |page|
+  page.name "Name this page"
+  page.url "<%= name %>"
+  # page.layout :layout_name
```

Comment:
That was intentional. Layouts are optional and I didn't want to introduce more arguments to the generator, so I just left a comment in there so it can be easily set if you want...